composer/loggers/remote_uploader_downloader.py

```diff
@@ -707,3 +708,7 @@ def upload_file(retry_index: int = 0):
         time.sleep(local_rank * local_rank_stagger)
 
         upload_file()
+
+    for destination in logger.destinations:
+        if isinstance(destination, MosaicMLLogger):
+            destination.log_metadata({'checkpoint_uploaded_time': time.time()})
```

Comment:
is this tracking what you are actually trying to track? It will simply overwrite every time a new file is uploaded.

Reply:
we only want last one to measure goodput